src/web/core/src/main/java/org/geoserver/web/wicket/EnvelopePanel.java

```diff
@@ -115,6 +121,35 @@ void initComponents() {
         add(crsContainer);
     }
 
+    private void addBoundingBoxValidators() {
+        minXInput.add(validatable -> validateAxis(validatable, maxXInput, "X"));
+
+        minYInput.add(validatable -> validateAxis(validatable, maxYInput, "Y"));
+
+        if (is3D()) {
+            minZInput.add(validatable -> validateAxis(validatable, maxZInput, "Z"));
+        }
+    }
+
+    /**
+     * Validate if bounding box min value is lower then max value for specific axis
+     *
+     * @param validatable
+     * @param field
+     * @param axis
+     */
+    private void validateAxis(
+            IValidatable<Double> validatable, DecimalTextField field, String axis) {
+        Double convertedValue = converter.convertToObject(field.getValue(), field.getLocale());
+        if (convertedValue != null && validatable.getValue() >= convertedValue) {
+            ValidationError error = new ValidationError();
+            error.setMessage(
+                    new ParamResourceModel("validation.boundingBoxAxisNonPositive", this, axis)
+                            .getObject());
+            validatable.error(error);
+        }
```

Comment:
Found a weird thing while interactively testing the change. The error message appears twice when I save, even if only one axis had invalid coordinates (and indeed the message is the same):

![image](https://user-images.githubusercontent.com/325433/219607429-90d7a206-cda7-4909-8ea1-695aac45290c.png)

Can you debug and figure out why the message is added twice? I was expecting to see it also in the tests below, but the count seems right there, e.g., 3 errors, 3 messages.

Also strange, if I  start clean and put the error on the Y axis, I'm still receiving a message for the X one: 

![image](https://user-images.githubusercontent.com/325433/219608001-c7af845e-7350-41e0-8397-8837d5d54125.png)

Oh... found it with a little debugging... the conversion in the validator is performed using the current locale, which for me is italian, where the "." is the thousands separator, not the decimal separator, so I get the wrong number. These fields are not meant to be edited in a locale-dependent way, the number conversion should be done using english locale.

Reply:
Some clarifiacation from me after debug. The values indeed wasn't converted right, but it works the same on English locale too, since by default, the input fields for bounding boxes use commas as decimal separator (for example when computing) which is invalid for some locales like English and Italian for example and valid for others like Russian and French (it was working fine for me because I forgot to switch from Russian). When panel is created using constructor the fields will have dots as separator (until computed). Plus user can actually input value with dots in those fields. From my point of view (knowing almost nothing about project internals) if input should work with english locale then it shouldn't be allowed to use comma (computed result with dot plus replacing comma with dot when input is submitted for example). But for now I'll just parse both cases.

Reply:
So I checked and if using dot there's another validation after that which doesn't allow that.